pyproject.toml

```diff
@@ -54,15 +54,13 @@ protobuf = "3.20.3"
 
 # Deployment
 boto3 = "^1.23.5"
-fastapi = "^0.93.0"
+fastapi = "^0.103.1"
 uvicorn = "^0.21.0"
-python-multipart = "^0.0.6"
 tqdm = "^4.64.1"
 transformers = "^4.30.2"
 certifi = "2023.07.22"
-cryptography = "41.0.2"
```

Comment:
@fd0r do you confirm that this is not useful anymore ?

Reply:
This was set to avoid a vulnerability, that means that the package that required it updated the version on their side, that good!